src/Microsoft.ML.Data/Transforms/NopTransform.cs

```diff
@@ -123,10 +123,11 @@ public RowCursor GetRowCursor(IEnumerable<Schema.Column> columnsNeeded, Random r
         public RowCursor[] GetRowCursorSet(IEnumerable<Schema.Column> columnsNeeded, int n, Random rand = null)
             => Source.GetRowCursorSet(columnsNeeded, n, rand);
 
-        public Func<int, bool> GetDependencies(Func<int, bool> predicate)
-        {
-            return predicate;
-        }
+        /// <summary>
+        /// Given a set of columns, return the input columns that are needed to generate those output columns.
+        /// </summary>
+        IEnumerable<Schema.Column> IRowToRowMapper.GetDependencies(IEnumerable<Schema.Column> dependingColumns)
+        => dependingColumns;
```

Comment:
If you look few lines above we have example of different style for =>.
Can we put tab here? (not tab, 4 spaces, we are not barbarians)